lxd/seccomp.go

```diff
@@ -445,6 +446,57 @@ func NewSeccompServer(d *Daemon, path string) (*SeccompServer, error) {
 	return &s, nil
 }
 
+func taskUidGid(pid int) (error, int32, int32) {
```

Comment:
Probably missing something obvious here, but how is Uid/Gid from /status different from the owner of the proc entry itself?

I've always just relied on the ownership of `/proc/PID` being correct to get uid/gid without needing any parsing.

Reply:
The ownership of /proc/<pid> is usually right but depends on e.g. the dumpability attribute being set to 1. If it isn't everything will be owned by root. I wasn't sure how the calling task in another procfs mount would be affected but I wouldn't want to rely on that.